src/dfmpreview/src/DfmProcess.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace Dfm_test
```

Comment:
Yes, It has been renamed in the new version!

---
In reply to: [73456951](https://github.com/dotnet/docfx/pull/558#discussion_r73456951) [](ancestors = 73456951)